tests/test_wkhtmltox.py

```diff
@@ -0,0 +1,44 @@
+import os
+
+from .utilities import (
+    PDF, assert_images_are_identical, assert_pdf_objects_are_identical
+)
+
+
+def test_convert_to_jpg():
+    import wkhtmltox
+    img = wkhtmltox.Image()
+    test_file = b'test.jpg'
+    img.set_global_setting(b'out', test_file)
+    img.set_global_setting(b'in', b'tests/simple.html')
+    img.convert()
+
+    assert os.path.isfile(test_file)
+    with open('tests/simple.jpg', 'rb') as org, open(test_file, 'rb') as new:
+        assert assert_images_are_identical(
+            'test_image', org.read(), new.read(), 'jpg')
+    try:
+        os.remove(test_file)
+    except OSError:
+        pass
```

Comment:
if the assertion fails, the test file remains and it's not deleted.

Reply:
:facepalm: